core/src/main/resources/org/apache/spark/ui/static/webui.css

```diff
@@ -357,6 +357,10 @@ a.expandbutton {
   width: 180px;
 }
 
+.executor-jvm-metrics-checkbox-div {
```

Comment:
Do we still need this oafter https://issues.apache.org/jira/browse/SPARK-33562 ?

Reply:
> Do we still need this oafter https://issues.apache.org/jira/browse/SPARK-33562 ?

Yea, need this, without this will look like 
![image](https://user-images.githubusercontent.com/46485123/101137094-3aaf1f80-3649-11eb-96db-5a498343e35e.png)

Reply:
Your branch is not up-to-date

Reply:
Oh, yea,   can remove